db/schema.rb

```diff
@@ -34,7 +34,7 @@
   add_index "builds", ["project_id"], name: "index_builds_on_project_id", using: :btree
 
   create_table "commands", force: :cascade do |t|
-    t.text     "command",    limit: 10485760
+    t.text     "command",    limit: 16777215
```

Comment:
Please also revert the changes to the limits that are scattered through schema.rb.  Depending on the DB configuration of the host where you ran the migrations, these limits are different.

If the migrations you added didn't change a table in this PR, we shouldn't see any changes to the table in schema.rb. Sorry if I wasn't clear in my earlier comment.